deb/changes.go

```diff
@@ -195,11 +195,6 @@ func (c *Changes) PackageQuery() PackageQuery {
 			}
 		}
 
-		ddebQuery = &AndQuery{
```

Comment:
Doesn't removing filtering by `c.Source` potentially select the wrong packages?

Reply:
We already have `<pkg name>-dbgsym` rule for each specific package, and it's not like multiple source packages should produce the same binary package. `binaryQuery` doesn't have `c.Source` rule, and it's constructed essentially the same way as `ddebQuery`.